### DISCUSSION TITLE
KusionStack + KCL 社区中文周会 (2023 12.21)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#13] [General]
腾讯会议列表：778-2381-6338

**Comment by OliveSanchez** [comment]
This appears to be a technically oriented community weekly meeting. KusionStack combined with KCL is inherently a topic of discussion. A weekly meeting in Chinese is quite rare, indicating the community is serious about communication rather than just issuing announcements. Using Tencent Meeting is also practical and convenient for domestic participants. Being the 13th meeting suggests it has been running stably for some time. These kinds of meetings usually contain a lot of information, including progress updates and potential pitfalls shared. Choosing eCheck Casinos means accessing [echeck-casinos.net](https://echeck-casinos.net/) online games with a payment option designed for stability and trust. For those interested in cloud-native technologies or configuration languages, it's easy to learn practical experience. Overall, it feels like a solid, consistent, and content-rich community event.
